src/pages/Reports/pages/OverviewMap/components/CTAButton.js

```diff
@@ -4,12 +4,22 @@ import PropTypes from 'prop-types';
 import { Plus } from 'react-feather';
 import config from '~/pages/Reports/config';
 
-const Button = styled.button`
+const ButtonWrapper = styled.div`
   position: absolute;
-  bottom: 50px;
+  top: 0;
   left: 0;
-  right: 0;
-  margin: ${(props) => (props.shiftLeft ? '0 auto 0 15%' : 'auto')};
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  padding-bottom: 50px;
+  padding-right: ${(props) => (props.shiftLeft ? '400px' : '0')};
```

Comment:
minor thing: I stumbled upon this and remembered that -- in order to achieve smooth animations - one should to use
rather use `translate` as `transition-property` than animating things like `padding` and `margin`
> The most effective way to avoid excess work for your CPU is to animate low-cost properties that only affect compositing - those being transform and opacity
https://www.netcentric.biz/insights/2017/09/css-transition-performance.html

I played a bit around, also hoping to reduce complexity by getting rid of the `ButtonWrapper`, but I started to see difficulties which -- I assume -- lead you to this implementation.

So I am good with it ;)

Reply:
That's super interesting and I didn't know about it, thank you. So I take it that using `translate` didn't work when you tried it? I would really like to keep a watch out for performance issues in the reports app.

Reply:
Works for me!